packages/core/src/modules/credentials/formats/jsonld/JsonLdCredentialFormatService.ts

```diff
@@ -229,9 +222,9 @@ export class JsonLdCredentialFormatService implements CredentialFormatService<Js
 
     const options = credentialRequest.options
 
-    if (options.challenge || options.domain || options.credentialStatus) {
+    if (options.challenge || options.domain || options.credentialStatus || options.created) {
       throw new AriesFrameworkError(
-        'The fields challenge, domain and credentialStatus not currently supported in credential options '
+        'The fields challenge, domain, created and credentialStatus not currently supported in credential options'
```

Comment:
Might be nice in the error, or log, which options were supplied that are not supported. Can save some time with debugging.

Reply:
Hmm makes sense. This was the cleanest way I could come up with to show which fields were the issue, without adding 4 if statements (which may be fine??). Let me know if you have a better approach:

```ts
    // Get a list of fields found in the options that are not supported at the moment
    const unsupportedFields = ['challenge', 'domain', 'credentialStatus', 'created'] as const
    const foundFields = unsupportedFields.filter((field) => options[field])

    if (foundFields) {
      throw new AriesFrameworkError(
        `The fields ${foundFields.join(', ')} are not currently supported in credential options`
      )
    }
```

Reply:
I would slightly rephrase error message to adapt a bit better for the case where is a single unsupported field: `Some fields are not currently supported in credential options: ${foundFields.join(', ')}`

Reply:
updated it to your suggestion @genaris

Reply:
I think the function you used is okay @TimoGlastra (I tried to do it with Object.keys but it did not change much) and I agree with the comment from @genaris